docs/_components/footer.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Footer
+---
+
+Check out the slim and medium footer examples on the [layout page](/layout#page-layouts).
```

Comment:
These links aren't quite working. May need to use some kind of site root variable in front of the `/`

Reply:
ok, added the baseurl - but we'll check on the federalist build